### ISSUE TITLE
redundant messaging

### ISSUE THREAD

**Issue by Dieterbe** [raintank/raintank-collector#26]
![event-redundant](https://cloud.githubusercontent.com/assets/20774/9812519/99832174-5886-11e5-8dc6-94dd634a9f9d.png)

first message looks good to me,
but the second doesn't have to be anything more than `dns OK`. the colon and all 3 words after are all redundant.

in other words I think 

```
<monitortype> <status>
<monitortype> <status>: <error message>
```

is all that is needed

**Comment by Dieterbe** [comment]
@mattttt what do you think?

**Comment by Dieterbe** [comment]
also, should we make these messages look as much like the subjects of our alert emails as possible?

**Comment by mattttt** [comment]
I agree that the "Monitor now OK" is redundant. Are there any situations where an OK message will include any other text?
